### ISSUE TITLE
Darker Button colour "+ Challenge"

### ISSUE THREAD

**Issue by harrygfox** [emfoundation/ce100-app#745]
## #40667e

The colour above is used for the border and text of the add challenge button found on a profile
See:
https://app.zeplin.io/project.html#pid=589c772f8ad534321d652f04&sid=589c95d3c4ac70861db5c611

This is not reflected in the code from PR #732 

![screen shot 2017-03-01 at 10 54 44](https://cloud.githubusercontent.com/assets/7805691/23456962/8b4e1a1c-fe6d-11e6-8179-a87d3fd81fd3.png)

**Comment by mk4111** [comment]
@harrygfox - you're right! I'll make a new PR to fix this after #732 is merged in! Hope that's ok!

**Comment by mk4111** [comment]
![screenshot from 2017-03-01 11-37-19](https://cloud.githubusercontent.com/assets/13470325/23458267/78c037ee-fe73-11e6-9041-0dbede90f230.png)

**Comment by mk4111** [comment]
@harrygfox - if happy, could you please close? Tank tank!
